src/main/utils/iconsPaths.ts

```diff
@@ -1,42 +1,37 @@
 import * as path from 'path';
-import { PackageType } from '../types/projectType';
 import { ScanUtils } from './scanUtils';
 
 export class IconsPaths {
     // Icons severities
-    static readonly NORMAL_SEVERITY: string = IconsPaths.getIconPath('normal');
-    static readonly PENDING_SEVERITY: string = IconsPaths.getSvgIconPath('Unknown');
-    static readonly UNKNOWN_SEVERITY: string = IconsPaths.getSvgIconPath('Unknown');
-    static readonly NOT_APPLICABLE_UNKNOWN_SEVERITY: string = IconsPaths.getSvgIconPath('notApplicableUnknown');
-    static readonly NOT_APPLICABLE_LOW_SEVERITY: string = IconsPaths.getSvgIconPath('notApplicableLow');
-    static readonly NOT_APPLICABLE_MEDIUM_SEVERITY: string = IconsPaths.getSvgIconPath('notApplicableMedium');
-    static readonly NOT_APPLICABLE_HIGH_SEVERITY: string = IconsPaths.getSvgIconPath('notApplicableHigh');
-    static readonly NOT_APPLICABLE_CRITICAL_SEVERITY: string = IconsPaths.getSvgIconPath('notApplicableCritical');
-    static readonly INFORMATION_SEVERITY: string = IconsPaths.getSvgIconPath('Low');
-    static readonly LOW_SEVERITY: string = IconsPaths.getSvgIconPath('Low');
-    static readonly MEDIUM_SEVERITY: string = IconsPaths.getSvgIconPath('Medium');
-    static readonly HIGH_SEVERITY: string = IconsPaths.getSvgIconPath('High');
-    static readonly CRITICAL_SEVERITY: string = IconsPaths.getSvgIconPath('Critical');
+    static readonly NORMAL_SEVERITY: string = IconsPaths.getIconPath(path.join('severities', 'normal'));
```

Comment:
To be aligned with all image file names, can we rename "normal.png" to "Normal.png"?